### PR TITLE
(phi0474.phi006_phi0474.phi011)

### PR DESCRIPTION
**Pull request by AlisonBabeu**
Continuing work on Cicero
Minor edits to previous files.

### CODE REVIEW

File: data/phi0474/phi007/__cts__.xml [hunk 1]
```diff
@@ -1,18 +1,19 @@
...
     
     <ti:edition urn="urn:cts:latinLit:phi0474.phi007.perseus-lat2" workUrn="urn:cts:latinLit:phi0474.phi007" xml:lang="lat">
         <ti:label xml:lang="lat">Pro Fonteio</ti:label>
-        <ti:description xml:lang="mul">Cicero. M. Tulli Ciceronis. Orationes, Vol. VI. Clark, Albert Curtis, editor. Oxford: Clarendon Press, 1911.</ti:description>
+        <ti:description xml:lang="mul">Cicero. M. Tulli Ciceronis. Orationes, Volume 6. Clark, Albert Curtis, editor. Oxford: Clarendon Press, 1911.</ti:description>
     </ti:edition>
 
     
     <ti:translation urn="urn:cts:latinLit:phi0474.phi007.perseus-eng2" workUrn="urn:cts:latinLit:phi0474.phi007" xml:lang="eng">
         <ti:label xml:lang="eng">For Marcus Fonteius</ti:label>
```
Comment (lcerrato) [inline data/phi0474/phi007/__cts__.xml:14]: This title is different from the English file title

File: data/phi0474/phi006/__cts__.xml [hunk 2]
```diff
@@ -1,25 +1,24 @@
...
-            Cicero, Marcus Tullius, creator; Yonge, Charles Duke, 1812-1891, translator
-        </ti:description>
+<ti:work xml:lang="lat" xmlns:ti="http://chs.harvard.edu/xmlns/cts"
+    urn="urn:cts:latinLit:phi0474.phi006" groupUrn="urn:cts:latinLit:phi0474">
+    <ti:title xml:lang="lat">Pro M. Tullio</ti:title>
+
+
+    <ti:translation urn="urn:cts:latinLit:phi0474.phi006.perseus-eng2"
+        workUrn="urn:cts:latinLit:phi0474.phi006" xml:lang="eng">
+        <ti:label xml:lang="eng"> For Marcus Tullius</ti:label>
```
Comment (lcerrato) [inline data/phi0474/phi006/__cts__.xml:10]: The English title is different from the file title.

**Comment by lcerrato** [comment]
@AlisonBabeu 
The title inconsistencies in Cicero are a bear. I don't think anyone ever checked what the translators used versus what Perseus was using.

File: data/phi0474/phi008/__cts__.xml [hunk 3]
```diff
@@ -1,19 +1,17 @@
...
     </ti:edition>
    
     
     <ti:translation urn="urn:cts:latinLit:phi0474.phi008.perseus-eng2" workUrn="urn:cts:latinLit:phi0474.phi008" xml:lang="eng">
-        <ti:label xml:lang="eng">For Aulus Caecina, The orations of Marcus Tullius Cicero Vol. 2 Three orations on the Agrarian law, the four against Catiline, the orations for Rabirius, Murena, Sulla, Archias, Flaccus, Scaurus, etc
-        </ti:label>
-        <ti:description xml:lang="eng">
-            Cicero, Marcus Tullius, creator; Yonge, Charles Duke, 1812-1891, translator
-        </ti:description>
+        <ti:label xml:lang="eng">For Aulus Caecina</ti:label>
```
Comment (lcerrato) [inline data/phi0474/phi008/__cts__.xml:12]: Two different English titles here.

File: data/phi0474/phi009/__cts__.xml [hunk 4]
```diff
@@ -1,19 +1,20 @@
...
     <ti:edition urn="urn:cts:latinLit:phi0474.phi009.perseus-lat2" workUrn="urn:cts:latinLit:phi0474.phi009" xml:lang="lat">
         <ti:label xml:lang="lat">De Imperio Cn. Pompei Ad Quirites</ti:label>
-        <ti:description xml:lang="mul">Cicero. M. Tulli Ciceronis. Orationes, Vol. I. Clark, Albert Curtis, editor. Oxford: Clarendon Press, 1908.</ti:description>
+        <ti:description xml:lang="mul">Cicero. M. Tulli Ciceronis. Orationes, Volume 1. Clark, Albert Curtis, editor. Oxford: Clarendon Press, 1908.</ti:description>
+        
     </ti:edition>
   
     
     <ti:translation urn="urn:cts:latinLit:phi0474.phi009.perseus-eng2" workUrn="urn:cts:latinLit:phi0474.phi009" xml:lang="eng">
         <ti:label xml:lang="eng">On Pompey’s Command</ti:label>
```
Comment (lcerrato) [inline data/phi0474/phi009/__cts__.xml:15]: Two different English titles

File: data/phi0474/phi010/__cts__.xml [hunk 5]
```diff
@@ -1,15 +1,16 @@
 <?xml version="1.0" encoding="UTF-8"?>
 <ti:work xml:lang="lat" xmlns:ti="http://chs.harvard.edu/xmlns/cts" groupUrn="urn:cts:latinLit:phi0474" urn="urn:cts:latinLit:phi0474.phi010" projid="latinLit:phi010">
-    <ti:title xml:lang="eng">For Aulus Cluentius</ti:title>
+    <ti:title xml:lang="lat">Pro A. CLuentio</ti:title>
```
Comment (lcerrato) [inline data/phi0474/phi010/__cts__.xml:3]: Capital L

File: data/phi0474/phi010/phi0474.phi010.perseus-lat2.xml [hunk 6]
```diff
@@ -4,12 +4,9 @@
 	<teiHeader xml:lang="eng">
 		<fileDesc>
 			<titleStmt>
-				<title>In Defense of Aulus Cluentius Habitus</title>
-				<title>Pro Sex. Roscio, De imperio Cn. Pompei, Pro Cluentio, In Catilinam, Pro
-					Murena, Pro Caelio</title>
-				
-				<author n="M. Tullius Cicero">M. Tullius Cicero</author>
-				<editor role="editor" n="OCT">Albert Clark</editor>
+				<title xml:lang="lat">Pro A. CLuentio Oratio</title>
```
Comment (lcerrato) [inline data/phi0474/phi010/phi0474.phi010.perseus-lat2.xml:7]: Different Latin titles and capitalized L in name

**Review by lcerrato — CHANGES_REQUESTED** [review]

File: data/phi0474/phi009/__cts__.xml (hunk 4 above)
Comment (AlisonBabeu) [reply data/phi0474/phi009/__cts__.xml:15]: I think that is a mistake here, thanks for catching.

**Comment by AlisonBabeu** [comment]
I had noticed the alternate Latin titles in a few cases but stuck with the
ones in the books we were using. I have the catalog data to help me keep
track of volumes and page links so I think I"m finally getting the hang of
this!

**Pull request merged**